sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -431,6 +432,11 @@ case class Explode(child: Expression) extends ExplodeBase {
     copy(child = newChild)
 }
 
+object Explode extends SupportsNamedArguments {
```

Comment:
Why should the 'explode' function support named arguments if it only has one argument?
```
explode(expr)
```

Reply:
Also, for a table function like `stack(n, expr1, ..., exprk)`, how would you add named arguments for it?

Reply:
Right now, we don't really have support for named arguments for final variadic argument. We can actually include support for it if needed. I think it just probably requires a bit of extra logic to handle the case. @dtenedor What do you think?